packages/components/forms/package.json

```diff
@@ -7,9 +7,11 @@
   },
   "dependencies": {
     "@contentful/f36-core": "^4.58.0",
+    "@contentful/f36-density-container": "^4.0.0-alpha.3",
```

Comment:
I was too late to recommend installing this as dev-dependency instead since it is not required for the package to be functional?